README.md

```diff
@@ -78,14 +113,24 @@ Here's a breakdown of all the available configuration items:
 
 #### A note on colouring
 
-* The card is hardcoded to display plunge pricing (e.g, below 0p/kWh) as blue
-* If the price is above `highlimit`, then the row is in red
-* If the price is above `mediumlimit`, then the row is coloured orange/yellow
-* Otherwise, the row is coloured is green.
-* These are reversed if `exportrates` is set to `true`
+* The card is hardcoded to display plunge pricing (e.g, below 0p/kWh) as blue. 
+* If the price is above `highLimit`, then the row is in red
+* If the price is above `mediumLimit`, then the row is coloured orange
+* if the price is above `lowLimit`, then the row is coloured dark green
+* if the price is below 'lowLimit', then the row is coloured green
+* These are reversed if `exportrates` is set to `true` (export rates have only 3 colours, red, orange and green)
+* Cheapes rate is coloured in light green / light blue
+* If Target Rate entity is included in the config, the target hours will be highlited in Navy Blue
+
+#### Screenshots
+![screenshot_1](assets/import.png)
+![screenshot_2](assets/export.png)
+
+Import rates with the Target Rates entity specified:
+
```

Comment:
e.g. I think if we made this a new heading and included your full yaml here, as a "Advanced Configurations" section?